javascript/example-customer-account--pre-auth--ts/extensions/customer-account-pre-auth-loyalty/shopify.extension.toml

```diff
@@ -0,0 +1,17 @@
+api_version = "unstable"
+
+[[extensions]]
+type = "ui_extension"
+name = "customer-account-pre-auth-loyalty-js"
+handle = "pre-auth-loyalty-extension-js"
+
+# [START config.setup-targets]
+[[extensions.targeting]]
+module = "./src/BlockLoyaltyExtension.ts"
+target = "customer-account.order-status.block.render"
+# [END config.setup-targets]
+
+# [START config.api-access]
+[extensions.capabilities]
+# api_access = true
+# [END config.api-access]
```

Comment:
It doesn't look like you actually use this tag or setting anywhere in the tutorial. I think this should be safe to delete

Reply:
updated~